config/compute/config.go

```diff
@@ -399,10 +399,6 @@ func Configure(p *config.Provider) { //nolint: gocyclo
 		config.MarkAsRequired(r.TerraformResource, "region")
 	})
 
-	p.AddResourceConfigurator("google_compute_region_per_instance_config", func(r *config.Resource) {
```

Comment:
Does this resource no longer exist? Is this unnecessary?

Reply:
I deleted this statement because this configuration does not have an effect. The `region` field is generated as a `Reference` field, and we mark the reference fields as `Optional` regardless of their status in the scheme. Also, as can be seen, even though the configuration was deleted, no diff occurred after generation. This shows that this configuration block has no function.

Reply:
Thank you for the explanation.